test/views/templates/ProviderInfoTemplateViewSpec.scala

```diff
@@ -64,9 +64,7 @@ class ProviderInfoTemplateViewSpec extends ViewBaseSpec {
       document.select("ul > li").get(4).text() shouldBe testMessages.viewLiabilities
       document.select("ul > li").get(5).text() shouldBe testMessages.viewPayments
 
-      document.select("p").get(3).text() shouldBe testMessages.accessibilityFeatures(
-        List(COGNITIVE, HEARING, MOTOR, VISUAL).map(_.toString.toLowerCase())
-      )
+      document.select("p").get(3).text() shouldBe "This software includes blindness or impaired vision, deafness or impaired hearing, motor or physical difficulties and cognitive impairments accessibility features."
```

Comment:
I think you could keep the  `testMessages.accessibilityFeatures` that was here before. But instead of the list of enums, have the required strings as fields in `testMessages`. Then use the same ones on lines 114 and 143